# Fei Zhao

âœ‰ï¸ [hi@kaopubear.top](mailto:hi@kaopubear.top)

ğŸ”— [linkedin.com/in/feizhao/](https://www.linkedin.com/in/feizhao/)

## SUMMARY

**Newly graduated PhD with background in bioinformatics seeking for a research & development position to develop and employ computational tools to analyze biological data.**

- 6-year experience of analyzing high throughput data, including data cleaning, functional annotation, normalization, analysis, interpretation, and visualization.
- Good management and communication skills exercised from organizing public outreach.

## EDUCATION

**University of Chinese Academy of Sciences** (2015.09-2021.07)
Ph.D. in Bioinformatics

- Research focus: functional genomics and epigenomics
- Developing methods, tools and platformsÂ for high-throughput sequencing-aided gene mapping
- **Awards**: 
    - Bank of China Achievement Award Scholarship (2021)
    - Pacemaker to Merit Student (2020-2021)
    - Excellent Student Cadre (2015-2016)

**Shandong Agricultural University** (2011.09-2015.07)
B.S. in Horticulture

## TECHNICAL SKILLS
**NGS Data Analysis**

- WholeÂ processÂ ofÂ dataÂ analysis: WGS, ChIP-Seq, RNA-Seq, PAT-Seq and WGBSÂ 
- Public database data mining: GEO, Ensembl

**Programming**

- R, Bash, SQL, Front-End, Git
- Workflow & environment management: Snakemake, Conda
- Operation and maintenance of linux servers and NAS
- Website development & interactive visualization
## PROFESSIONAL EXPERIENCE
**A one-stop multi-omics BSA-based workflow and polyploid species analysis platform.** (2019-now)

- Developed [Triti-Map](https://github.com/fei0810/Triti-Map), a Snakemake based workflow for identifying genetic markers associated with a mutant phenotype by combining with multi-omics BSA and _de novo_ assembly.
- Built an online platform for SNP annotation, orthologous gene analysis, and multiple genome syntenic comparisons and visualizations based on the R Shiny framework.
- Guided new graduate student with bioinformatics skills.

**A data-driven interface for retrieving upstream regulators from plant multi-omics data.** (2017-2020)

- Participated in the development and maintenance of [Plant Regulomics](http://bioinfo.sibs.ac.cn/plant-regulomics/).
- Collected and analyzed _Arabidopsis thaliana_ and rice multi-omics public data.Â 
- Developed and maintained some front-end and back-end modules of the online web platforms.Â 
- Over 24,000 users have already used the online platform.

**Jasmonate-mediated wound signaling promotes plant regeneration.** (2018-2019)

- Collaborated with students in biology who performed the experiments.
- Completed a multi-omics analysis, including histone methylation ChIP-Seq and time series transcriptome analysis, to reveal the genes of interest's epigenetic regulatory mechanisms. 
## TEACHING & PUBLICÂ OUTREACH

- **Podcast host of ç†Šè¨€ç†Šè¯­** (2020.03-now): Owning a Chinese [podcast](https://podcast.kaopubear.top/) aiming at bioinformatics related students and broader researchers from high-education organizations (> 90,000 plays).
- **SSPAI Senior Author** (2018.09-now): Wrote [articles](https://sspai.com/u/kaopubear/posts) on efficiency and productivity tools (> 927,000 read).
- **Teaching Assistant** (2017-2018) : *biostatistics*, University of Chinese Academy of Sciences
- **Lead organizer**:
   - Made "[Hands-On Training in RNA-seq Data Analysis](https://www.bilibili.com/video/BV1KJ411p7WN)" video tutorials, available online for freeï¼ˆ> 68,000 views, 2020.01);
   - Organized an online learning community, collected more than 600 learning materials (2019-2020);
   - Hosted and organized the 1st Bioinformatics Talent Development Forum, 2 days, 28 speakers and 100+ audiences (2019.07);
   - Initiated and participated in the [Bioinformatics Knowledge Base](https://www.yuque.com/biotrainee) project (> 185,000 visits, 2019.02).

## PUBLICATIONS

> \#, Co-first author;Â Â *, Corresponding author

- **Zhao, Fei**, Shilong Tian, Zijuan Li, Luhuan Ye, Yili Zhuang, Meiyue Wang, Yilin Xie, et al. (2022). Utility of Triti-Map for Bulk-Segregated Mapping of Causal Genes and Regulatory Elements in Triticeae. ***Plant Communications***, February, 100304. 10.1016/j.xplc.2022.100304.
- Zhao, Hong#, Yue Zhang#, Hui Zhang#, Yanzhai Song#, **Fei Zhao**#, Yuâ€™e Zhang#, Sihui Zhu#,  Hongkui Zhang, Zhendiao Zhou, Han Guo, Miaomiao Li, Junhui Li, Qiang Gao, Qianqian Han, Huaqiu Huang, Lucy Copsey, Qun Li, Hua Chen, Enrico Coen\*, Yijing Zhang\*, Yongbiao Xue\*. (2021). Origin, Loss and Regain of Self-Incompatibility in Angiosperms. ***The Plant Cell***, 10.1093/plcell/koab266.
- Xiaojuan Ran, Tengfei Tang, Meiyue Wang, Luhuan Ye, Yili Zhuang, **Fei Zhao**\*, and Yijing Zhang\*. (2021). CSCS: A Chromatin State Interface for Chinese Spring Bread Wheat. ***ABIOTECH***, May. 10.1007/s42994-021-00048-z
- Qiuhong Wu#, **Fei Zhao**#, Yongxing Chen, Panpan Zhang, Huaizhi Zhang, Guanghao Guo, Jingzhong Xie, Lingli Dong, Ping Lu, Miaomiao Li, Shengwei Ma, Tzion Fahima, Eviatar Nevo, Hongjie Li, Yijing Zhang\*, Zhiyong Liu\*. (2021). Bulked segregant CGT-Seq-facilitated map-based cloning of a powdery mildew resistance gene originating from wild emmer wheat (Triticum dicoccoides). **_Plant biotechnology journal_**, 10.1111/pbi.13609
- Xiaojuan Ran#, **Fei Zhao**#, Yuejun Wang#, Jian Liu, Yili Zhuang, Luhuan Ye, Meifang Qi, Jingfei Cheng, Yijing Zhang*. (2020) Plant Regulomics: A Data-driven Interface for Retrieving Upstream Regulators from Plant Multi-omics Data, **_The Plant Journal_**, 101(1):237-248
- Jing Pan#, **Fei Zhao**#, Guifang Zhang#, Yu Pan, Lijun Sun, Ning Bao, Peng Qin, Lyuqin Chen, Jie Yu*, Yijing Zhang*, Lin Xu*. (2019) Control of de novo root regeneration efficiency by developmental status of Arabidopsis leaf explants, **_Journal of Genetics and Genomics_**. 46(3):133-140
- Guifang Zhang#, **Fei Zhao**#, Lyuqin Chen#, Yu Pan, Lijun Sun, Ning Bao, Teng Zhang, Chun-Xiao Cui, Zaozao Qiu, Yijing Zhang and Lin Xu*. (2019) Jasmonate-mediated wound signalling promotes plant regeneration, **_Nature Plants_**. 5(5):491-497
- Zhenling Lv, Zijuan Li, Meiyue Wang, **Fei Zhao**, Wenjie Zhang, Changping Li, Lei Gong, Yijng Zhang\*, Annaliese S. Mason\* and Bao Liu\*. (2021). Conservation and trans-regulation of histone modification in the A and B subgenomes of polyploid wheat during domestication and ploidy transition. _**BMC biology**_, *19*(1), 1-16.
- Meiyue Wang#, Zijuan Li#, Yuâ€™e Zhang#, Yuyun Zhang#, Yilin Xie, Luhuan Ye, Yili Zhuang, Kande Lin, **Fei Zhao**, Jingyu Guo, Wan Teng, Wenli Zhang, Yiping Tong, Yongbiao Xue\*, Yijing Zhang\* (2021)An atlas of wheat epigenetic regulatory elements reveals subgenome-divergence in the regulation of development and stress responses, **_The Plant Cell_**
- jizeng Jia#, Yilin Xie#, Jingfei Cheng#, Chuizheng Kong#, Meiyue Wang,Lifeng Gao, **Fei Zhao**, Jingyu Guo, Kai Wang, Guangwei Li, Dangqun Cui, Tiezhu Hu, Guangyao Zhao\*, Daowen Wang\*, Zhengang Ru\* & Yijing Zhang\*(2021)Homology-mediated inter-chromosomal interactions in hexaploid wheat lead to specific subgenome territories following polyploidization and introgression, **_Genome Biology_**
- Jun Fang#, Fantao Zhang#, Hongru Wang, Wei Wang, **Fei Zhao**, Zijuan Li, Changhui Sun, Faming Chen,Fan Xu, Shuoqi Chang, Liang Wu, Qingyun Bu, Pingrong Wang, Jiankun Xie, Fan Chena, Xuehui Huang, Yijing Zhang, Xinguang Zhu, Bin Han, Xiaojian Deng, and Chengcai Chu* (2019) Ef-Cd Locus Shortens Rice Maturity Duration without Yield Penalty, **_Proceedings of the National Academy of Sciences of the United States of America_**,116(37):18717-18722
- Meifang Qi#, Zijuan Li#, Chunmei Liu#, Wenyan Hu, Luhuan Ye, Yilin Xie, Yili Zhuang, **Fei Zhao**, Wan Teng, Qi Zheng, Zhenjun Fan, Lin Xu, Zhaobo Lang, Yiping Tong\*, and Yijing Zhang\*. (2018) CGT-seq: epigenome-guided de novo assembly of the core genome for divergent populations with large genome, **_Nucleic Acid Research_**, gky522.
- Dongmei Yin#, Changmian Ji#, Xingli Ma#, Hang Li, Wanke Zhang, Song Li, Fuyan Liu, unkun Zhao, Fapeng Li, Ke Li, Longlong Ning, Jialin He, Yuejun Wang, **Fei Zhao**, Yilin Xie, Hongkun Zheng, Xingguo Zhang, Yijing Zhang, and Jinsong Zhang* (2018) Genome of an allotetraploid wild peanut Arachis monticola: a de novo assembly, **_GigaScience_**, 7(6)

> Lasted updated on 2022-3-5

